src/main/resources/de/numcodex/feasibility_gui_backend/terminology/terminology-systems.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
example could be part of resources - otherwise should be supplied as part of ontology

Reply:
done